doc/generate_logos.py

```diff
@@ -185,6 +195,53 @@ def convert_to_png(fn_source, output_dir, sizes):
                 logging.debug("command: %s" % cmd)
                 logging.debug("return code: %s" % p.returncode)
 
+def convert_to_png_win(fn_source, output_dir, sizes):
```

Comment:
What's the difference between these two functions. They look very similar. Can they be combined with only logic for the different parts split between Windows and non-Windows?